### PR TITLE
Solve dish issues

### PR DESCRIPTION
**Pull request by BotBw**
Solved some issues, see development

### CODE REVIEW

File: src/main/java/seedu/duke/controllers/DishController.java [hunk 1]
```diff
@@ -67,54 +67,82 @@ private void listDishes() {
...
-        double price = InputParser.getDouble("The price of dish: ");
-        try {
-            dishManager.addDish(new Dish(name, price));
-        } catch (IllegalArgumentException e) {
-            System.out.println("Please make sure the name is not empty and price is positive");
+        if (name.isEmpty()) {
+            System.out.println("Please make sure the name is not empty and price is non-negative");
+            return;
+        }
+        if (dishManager.isNameExist(name)) {
```
Comment (Lim-Shi-Han) [inline src/main/java/seedu/duke/controllers/DishController.java:136]: I think "is..." is only for naming boolean attributes, correct me if I am wrong though!

**Review by Lim-Shi-Han — COMMENTED** [review]
lgtm except the comment about naming

File: src/main/java/seedu/duke/controllers/DishController.java (hunk 1 above)
Comment (BotBw) [reply src/main/java/seedu/duke/controllers/DishController.java]: You are correct!
I renamed it already.

**Review by Lim-Shi-Han — APPROVED** [review]
lgtm if u r ready to merge I think can go ahead

**Pull request merged**